configure.ac

```diff
@@ -175,7 +175,8 @@ if test "$krb5" != no; then
    AC_CHECK_FUNC(res_send, :,
                  AC_CHECK_LIB(resolv, res_send, LIBZEPHYR_LIBS="-lresolv $LIBZEPHYR_LIBS",
                               AC_CHECK_LIB(resolv, __res_send, LIBZEPHYR_LIBS="-lresolv $LIBZEPHYR_LIBS",
-                                           AC_MSG_ERROR(Zephyr requires libresolv along with Kerberos V.))))
+                                           AC_CHECK_LIB(resolv, res_9_send, LIBZEPHYR_LIBS="-lresolv $LIBZEPHYR_LIBS",
```

Comment:
Take a look at how I did the check for [Hesiod](https://github.com/achernya/hesiod/blob/master/configure.ac#L39). I think it's better to explicitly #include the header rather than check for the explicit mangling.

Reply:
Thanks for the suggestion. I copied that logic, but tweaked it since libresolv is required here. (AFAIK Hesiod doesn't actually require libresolv? It just does `AC_MSG_RESULT(no)`.)